starter_code/src/movies.js

```diff
@@ -2,14 +2,54 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+const orderByYear=(Arr) =>{
+    let years=[]
+    let final=[]
+    Arr.forEach(elem=>{
+      if(years.indexOf(elem.year)===-1)
+      years.push(elem.year)
+    })
+    years.sort()
+    for(i=0;i<years.length;i++)
+    {
+     let moviesYear=Arr.filter(elem=>elem.year===years[i])
+        moviesYear.sort(function (a, b){
```

Comment:
Les faltó la condición del segundo (ordenar por titulo en caso de que tengan el mismo año). Pueden poner las dos en el mismo lugar y que vayan corriendo las condiciones por tiempo de ejecución.

`const orderByYear = Arr => {`
  `let years = []`
  `let final = []`
  `Arr.forEach(elem => {`
   ` if (years.indexOf(elem.year) === -1) years.push(elem.year)`
  `})`
  `years.sort()`
  `for (i = 0; i < years.length; i++) {`
    `let moviesYear = Arr.filter(elem => elem.year === years[i])`
    `moviesYear`
      `.sort(function(a, b) {`
       **` if (a.title > b.title) return 1`
        `if (a.title < b.title) return -1 `**
        `return b.title - a.title`
      `})`
      `.forEach(elem => final.push(elem))`
  `}`
  `return final`
`}`